src/js/components/CheckBox.js

```diff
@@ -54,13 +44,15 @@ export default class CheckBox extends Component {
           </svg>
         </span>
       </span>,
-      labelNode
+      label
```

Comment:
Now it is a label tag embedding a checkbox. See the other changes to the file (and the PR notes).